rust_build_utils/rust_utils.py

```diff
@@ -38,6 +39,9 @@ def __post_init__(self):
     def is_msvc(self):
         return self.rust_target.endswith("-msvc")
 
+    def add_features(self, features: List[str]):
```

Comment:
I think `add` implies that the features will be appended, here they are substituted, so I would use `set_features`